package.json

```diff
@@ -6,6 +6,7 @@
     "@testing-library/jest-dom": "^5.16.5",
     "@testing-library/react": "^13.4.0",
     "@testing-library/user-event": "^13.5.0",
+    "prop-types": "^15.8.1",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
```

Comment:
- [ ] Your MIT link is  in README leading to 404 page, kindly update it

Reply:
- [ ] Please remove all unnecessary image/logos that were created by CRA 
![image](https://github.com/MarcoDiaz2000/math-magicians/assets/62705947/cbc18652-668f-4c3e-97c3-ca922455b956)

Reply:
> * [ ]  Your MIT link is  in README leading to 404 page, kindly update it

Hi, the "prop-types": "^15.8.1", is the last version

![update](https://github.com/MarcoDiaz2000/math-magicians/assets/125866339/681f9623-b892-41b5-ab34-ae9f3c6c9758)

I will continue with the rest of the requested changes.

Thank you

Reply:
Ok, I understand, thank you